dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -24,16 +24,6 @@ class RegistrationsControllerTest < ActionController::TestCase
       age: '13',
       user_type: 'student'
     }
-
-    @default_params_new_signup = {
-      name: 'A name',
-      password: 'apassword',
-      email: 'an@email.address',
-      gender: 'F',
-      age: '13',
-      user_type: 'student',
-      new_sign_up: true
-    }
```

Comment:
This wasn't actually working because the params had to be passed in the format of `{new_sign_up: true, user: {...[user params]}}` rather than all flat together so it wasn't running any of the tests below in the new signup format. I've adjusted the tests to reflect this and added [a helper function](https://github.com/code-dot-org/code-dot-org/pull/61456/files#diff-0d9062d35e146dfe50438e6048de0069ae9dd5e0cddca6fe6a628e3bbb5e4e0eR908-R915) to run the `begin_sign_up` method which creates the `PartialRegistration` of a user for their email + password